### DISCUSSION TITLE
Viewing Selected Polygons

### DISCUSSION THREAD

**Discussion by jmarokhovsky** [opengeos/leafmap#834] [Q&A] [answered]
I'm using leafmap and solara to display a vector which highlights some selectable countries that correspond to the information I'm trying to display. I see that on hover an info widget appears with the properties of the polygon below the mouse, but I can't find the correct field to send to solara to let solara know the desired country.  Is it possible to get information on or some properties of the polygon which a user has clicked on?

**Comment by giswqs** [comment, marked as answer]
There is no direct way to get the polygon from mouse click events. One workaround is to observe the map mouse click events, and then get the clicked coordiantes to be used for intersecting a GeoDataFrame. 

https://github.com/opengeos/leafmap/blob/30e52c5dad305a14beb18bb4fc404fa9b4ea4cb9/leafmap/leafmap.py#L1236-L1245

**Reply by jmarokhovsky** [reply]
Thank you! That makes sense, I'll try that out
